packages/testring-typings/config.d.ts

```diff
@@ -14,5 +14,6 @@ export interface IConfig {
     tests: string,
     httpThrottle: number,
     loggerLevel: number,
+    envConfig: string | void,
```

Comment:
Unsure if this is right type for optional params

Reply:
use `envConfig?: string`